EF_Example/Program.cs

```diff
@@ -29,6 +32,18 @@
 
 var app = builder.Build();
 
+var context = new ExampleDbContext(connectionString);
+
+// can be deleted after its run 1s (does a migration for old passwords in a hacky way)
+var users = context.UserLogin.Where(w => !w.Password.StartsWith("$2b$10$")).ToArray();
+foreach (var user in users)
+{
+    user.Password = BCrypt.Net.BCrypt.HashPassword(user.Password, BCrypt.Net.BCrypt.GenerateSalt(10, SaltRevision.Revision2B));
```

Comment:
This is hacky but will update all current user passwords to use bcrypt in the db